cloud/cmd/cloudcore/app/server.go

```diff
@@ -143,7 +143,7 @@ func NegotiateTunnelPort() (*int, error) {
 	kubeClient := client.GetKubeClient()
 	err := httpserver.CreateNamespaceIfNeeded(kubeClient, constants.SystemNamespace)
 	if err != nil {
-		return nil, errors.New("failed to create system namespace")
+		return nil, fmt.Errorf("failed to create system namespace, %v", err.Error())
```

Comment:
LGTM overall.

Just a nit, if use `%v`, directly use `err` is enough.

Reply:
true, thanks :)